src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -1645,10 +1713,18 @@ public int update(ByteBuffer src, ByteBuffer dst) {
         }
 
         @Override
-        public int doFinal(byte[] in, int inOfs, int inLen, byte[] out, int outOfs) {
-            int len = gctr.doFinal(in, inOfs, inLen, out, outOfs);
-            ghash.doFinal(out, outOfs, len);
-            return len;
+        public int doFinal(byte[] in, int inOfs, int inLen, byte[] out,
+                           int outOfs) {
+            int len = 0;
+
+            if (inLen >= PARALLEL_LEN) {
+                len = implGCMCrypt(in, inOfs, inLen, out, outOfs, out, outOfs, gctr, ghash);
```

Comment:
nit: exceeds 80 chars

Reply:
The editor keeps reverting these for some reason.. it's very annoying